tox.ini

```diff
@@ -27,6 +27,8 @@ deps =
     #git+https://github.com/manahl/pytest-plugins.git@54ca3c0#egg=pytest-verbose-parametrize&subdirectory=pytest-verbose-parametrize
     pytest-xdist<1.28.0  # pyup: < 1.28 # blocked by `pytest`
     shade==1.31.0
+    # https://github.com/ansible/molecule/issues/1724
+    selinux; platform_system=='Linux'
```

Comment:
Concerns:

* https://github.com/ssbarnea/selinux/blob/master/setup.py#L32
* https://github.com/ssbarnea/selinux/issues/1
* Missing some setuptools configuration like `python_requires` etc.

Can you give some guarantees on this package before we adopt it?